UC4-chaincode/src/test/resources/test_configs/exam_result_contract/AddExamResultTestIO.json

```diff
@@ -425,6 +511,20 @@
           ]
         }
       ],
+      "examContract" : [
+        "ExampleCourse:M.1:Written Exam:2021-02-12T10:00:00",
+        {
+          "examId": "ExampleCourse:M.1:Written Exam:2021-02-12T10:00:00",
```

Comment:
examIds are not consistent with the API definition (should not cause any problems with the current implementation though).